ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__azure-4.17-nightly-multi.yaml

```diff
@@ -0,0 +1,52 @@
+build_root:
+  image_stream_tag:
+    name: ci-tools-build-root
+    namespace: ci
+    tag: latest
+images:
+- dockerfile_path: prow/Dockerfile
+  to: ocp-qe-perfscale-ci-azure-417-multi
+promotion:
+  to:
+  - namespace: ci
+    tag: latest
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.17"
+  multi-latest:
+    candidate:
+      architecture: multi
+      product: ocp
+      stream: nightly
+      version: "4.17"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- always_run: false
+  as: control-plane-multi-ipsec-3nodes
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: azure-qe
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "3"
+      COMPUTE_NODE_TYPE: Standard_D4ps_v5
+      CONTROL_PLANE_INSTANCE_TYPE: Standard_D8ps_v5
+      CUSTOM_AZURE_REGION: centralus
+      OCP_ARCH: arm64
```

Comment:
I don't think we need this, but I may miss something, pls let me know why this is added if you think it is needed.

Reply:
I found it here https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/conf/azure/ipi-conf-azure-ref.yaml#L44-L47. We can use the default amd64 for control plane.
```
- name: OCP_ARCH
    default: "amd64"
    documentation: |-
      The architecture of the control plane nodes (e.g., amd64, arm64).
```